sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -441,4 +441,8 @@ class DataFrameSuite extends QueryTest {
     checkAnswer(df.select(df("key")), testData.select('key).collect().toSeq)
   }
 
+  test("show") {
+    testData.select($"*").show()
```

Comment:
Yes, I realized it is not good, it's not checking any answer. I will modify it later

BR,
Jacky

- ·¢×Ô Smartisan T1 -

2015Äê2ÔÂ26ÈÕ£¬ÏÂÎç2:51ÓÚ Reynold Xin <notifications@github.com</<mailto:notifications@github.com>

In sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala<https://github.com/apache/spark/pull/4767#discussion_r25408008>:

> @@ -441,4 +441,8 @@ class DataFrameSuite extends QueryTest {
>      checkAnswer(df.select(df("key")), testData.select('key).collect().toSeq)
>    }
>
> +  test("show") {
> +    testData.select($"*").show()


actually this is a bad idea. it prints a lot of garbage to console. maybe we can ignore the test and add a comment saying this is ignored but we want to make sure it compiles correctly. do the same for java

¡ª
Reply to this email directly or view it on GitHub<https://github.com/apache/spark/pull/4767/files#r25408008>.